CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+faucet-pipeline-core version history
+====================================
+
+
+v1.1.0
+------
+
+_2018-10-23_
+
+significant changes for end users:
```

Comment:
_significant_ might be overstating it, but I'd like this terminology to be consistent across versions